src/bidiMapper/domains/network/NetworkStorage.ts

```diff
@@ -291,32 +287,14 @@ export class NetworkStorage {
 
   /** #@see https://w3c.github.io/webdriver-bidi/#get-the-network-intercepts */
   getNetworkIntercepts(
-    event: Exclude<
-      ChromiumBidi.Network.EventNames,
-      ChromiumBidi.Network.EventNames.FetchError
-    >,
-    requestId: Network.Request
+    requestId: Network.Request,
+    phase: Network.InterceptPhase
   ): Network.Intercept[] {
     const request = this.#requestMap.get(requestId);
     if (!request) {
       return [];
     }
 
-    let phase: Network.InterceptPhase | undefined = undefined;
```

Comment:
This wasn't being used for anything else. It was just a literal implementation of https://w3c.github.io/webdriver-bidi/#network-intercepts